Soccar [Game-Server]/src/nl/soccar/gameserver/model/Chat.java

```diff
@@ -0,0 +1,84 @@
+package nl.soccar.gameserver.model;
+
+import nl.soccar.gameserver.controller.socnet.message.ChatMessage;
+import nl.soccar.gameserver.model.command.Command;
+import nl.soccar.gameserver.model.command.CommandDispatcher;
+import nl.soccar.gameserver.model.session.RoomWrapper;
+import nl.soccar.library.enumeration.Privilege;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * @author Lesley
+ */
+public final class Chat {
+
+    private final List<String> mutedNames = new ArrayList<>();
+
+    private final RoomWrapper room;
+
+    public Chat(RoomWrapper room) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)